src/BlockTypes/Checkout.php

```diff
@@ -411,4 +413,39 @@ protected function register_block_type_assets() {
 		$shared_chunks = [ 'cart-blocks/order-summary-shipping--checkout-blocks/order-summary-shipping-frontend' ];
 		$this->register_chunk_translations( array_merge( $chunks, $vendor_chunks, $shared_chunks ) );
 	}
+
+	/**
+	 * Get list of Checkout block & its inner-block types.
+	 *
+	 * @return array;
+	 */
+	public static function get_checkout_block_types() {
+		$block_types = [];
+
+		if ( Package::feature()->is_feature_plugin_build() ) {
+			$block_types[] = 'Checkout';
+			$block_types[] = 'Checkout';
```

Comment:
Checkout is twice here
```suggestion
			$block_types[] = 'Checkout';
```

Reply:
This is throwing an error for me.

Reply:
Thank you, @senadir! 💯